src/index.html

```diff
@@ -11,7 +11,185 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+    >
+      <fieldset
+        name="personal-info"
+        class="fieldset"
+      >
+        <legend>Personal information</legend>
+
+        <label class="form-field">
+          Surname:
+          <input
+            type="text"
+            autocomplete="off"
+            name="surname"
+            required
+          >
+        </label>
+
+        <label class="form-field">
+          Name:
+          <input
+            type="text"
+            autocomplete="off"
+            name="name"
+            required
+          >
+        </label>
+
+        <label class="form-field">
+          How old are You?
+          <input
+            type="number"
+            name="age"
+            required
+            min="1"
+            max="100"
+            placeholder="12"
```

Comment:
```suggestion
            value="12"
```
It should be a default value, not just placeholder
![image](https://user-images.githubusercontent.com/96409563/221858823-3dcb60ca-3215-4af1-9ef9-0c9eb920f7b5.png)